runtime/cfdumper/main.c

```diff
@@ -516,7 +516,20 @@ static void dumpMethod(J9CfrClassFile* classfile, J9CfrMethod* method)
 	j9tty_printf( PORTLIB, "  Signature: %i -> %s\n", method->descriptorIndex, classfile->constantPool[method->descriptorIndex].bytes);
 	j9tty_printf( PORTLIB, "  Access Flags: 0x%X ( ", method->accessFlags);
 	printModifiers(PORTLIB, method->accessFlags, INCLUDE_INTERNAL_MODIFIERS, MODIFIERSOURCE_METHOD);
-	j9tty_printf( PORTLIB, " )\n");
+#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
+	if (classfile->accessFlags & CFR_ACC_VALUE_TYPE) {
+		J9CfrConstantPoolInfo name = classfile->constantPool[method->nameIndex];
+		if (J9UTF8_LITERAL_EQUALS(name.bytes, name.slot1, "<vnew>")) {
+			for (int i = 0; i < classfile->attributesCount; i++) {
```

Comment:
This is not legal C code (variables cannot be declared in `for` loops); `i` is already declared on line 511.